sw/tools/control_effectiveness/config/quadrotor.json

```diff
@@ -0,0 +1,159 @@
+{
+  "variables" : {
+    "filt_cutoff" : 3.2,
+    "act_dyn" : 0.008
```

Comment:
I think here it would be nice to put the time constant in rad/s, and then do the transformation to discrete time with the first order hold in the script such that it is irrespective of sample frequency. Also, this could be a property of an actuator as they can have different dynamics.

Reply:
I agree, but then embedded should also do that to make configuration easier, not sure it is already the case